recipe/meta.yaml

```diff
@@ -1,18 +1,19 @@
+{% set version = "0.5.0" %}
+
 package:
   name: pyorc
-  version: "0.5.0"
+  version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/pyorc/pyorc-0.5.0.tar.gz
+  url: https://pypi.io/packages/source/p/pyorc/pyorc-{{ version }}.tar.gz
   sha256: 89c4c708c653d0fe90f251fbcc4a6057daf7cf63c23dfe3d644d9ed2603e296d
```

Comment:
This should probably be templated as well.

Reply:
Sure we can do that if you prefer. As this is the only place it is used, left it.

The URL specifically is important for the auto-update bot to scan & pick up new releases as they become available.